web/client/utils/__tests__/CoordinatesUtils-test.js

```diff
@@ -766,6 +766,18 @@ describe('CoordinatesUtils', () => {
         const uc = [2, 4];
         expect(makeBboxFromOWS(lc, uc)).toEqual([2, 2, 4, 4]);
     });
+    it('getCesiumBoundFromOWS passing extent', ()=>{
```

Comment:
to create a more grouped and readable tests you can wrap different it functions in a describe

```suggestion
     describe('getCesiumBoundFromOWS', ()=>{
    it('passing extent', ()=>{
```